src/display/annotation_layer.js

```diff
@@ -556,6 +565,99 @@ class LinkAnnotationElement extends AnnotationElement {
     }
     link.className = "internalLink";
   }
+
+  _bindResetFormAction(link, resetForm) {
+    if (!this._fieldObjects) {
+      warn(
+        `_bindResetFormAction - "resetForm" action not supported, ` +
+          "ensure that the `fieldObjects` parameter is provided."
+      );
+      return;
+    }
+
+    link.className = "internalLink";
+    const otherClickAction = link.onclick;
+    link.onclick = () => {
```

Comment:
Depending on the data, you may or may not have set either the href or className at this point. Furthermore, you also need to ensure that a dummy `onclick` is added in the fallback-case.

All-in-all, can you please make the following changes which I believe should cover all of the cases that could possibly happen here?

```suggestion
    const otherClickAction = link.onclick;
    if (!otherClickAction) {
      link.href = this.linkService.getAnchorUrl("");
    }
    link.className = "internalLink";
      
    if (!this._fieldObjects) {
      warn(
        `_bindResetFormAction - "resetForm" action not supported, ` +
          "ensure that the `fieldObjects` parameter is provided."
      );
      if (!otherClickAction) {
        link.onclick = () => false;
      }
      return;
    }
    
    link.onclick = () => {
```